src/test/java/org/cactoos/ScalarTest.java

```diff
@@ -23,27 +23,28 @@
  */
 package org.cactoos;
 
+import org.cactoos.scalar.NoNulls;
 import org.junit.Test;
 
 /**
- * Test case for {@link Scalar.NoNulls}.
+ * Test case for {@link NoNulls}.
  * @since 0.11
  * @checkstyle JavadocMethodCheck (500 lines)
  */
 public final class ScalarTest {
 
     @Test(expected = IllegalArgumentException.class)
     public void failForNullArgument() throws Exception {
-        new Scalar.NoNulls<>(null).value();
+        new NoNulls<>(null).value();
     }
 
     @Test(expected = IllegalStateException.class)
```

Comment:
@dgroup The same above.

Reply:
disagree, https://github.com/yegor256/cactoos/pull/990#discussion_r247960573